samples/drivers/espi/src/main.c

```diff
@@ -37,12 +37,37 @@ static struct espi_callback vw_rdy_cb;
 static struct espi_callback vw_cb;
 static struct espi_callback p80_cb;
 
+static u8_t espi_rst_sts;
+static void host_warn_handler(u32_t signal, u32_t status)
+{
+	switch (signal) {
+	case ESPI_VWIRE_SIGNAL_HOST_RST_WARN:
+		LOG_INF("Host reset warning %d", status);
+#ifndef CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE
+		LOG_INF("HOST RST ACK %d", status);
+		espi_send_vwire(espi_dev, ESPI_VWIRE_SIGNAL_HOST_RST_ACK,
+				status);
+#endif
+		break;
+	case ESPI_VWIRE_SIGNAL_SUS_WARN:
+		LOG_INF("Host suspend warning %d", status);
+#ifndef CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE
+		LOG_INF("SUS ACK %d", status);
+		espi_send_vwire(espi_dev, ESPI_VWIRE_SIGNAL_SUS_ACK, status);
+#endif
```

Comment:
Could this use `if (IS_ENABLED(CONFIG...)) {` instead?

Reply:
Done